entrypoint.py

```diff
@@ -43,14 +59,114 @@ def check_max_answer_allowed(self, series, json_response_df):
         return False
 
     def validate_uploaded_data(self, series, json_response_df):
-        return self.check_max_answer_allowed(series, json_response_df)
+        data_type_verification = self.check_data_type(series, json_response_df)
+        max_answer_verification = self.check_max_answer_allowed(series, json_response_df)
+        return data_type_verification or max_answer_verification
+
+    def split_and_clean(self, curr_list_answers, token=r'[,=]'):
+        curr_list_answers_list = list(
+            curr_list_answers.str.split(token).values)
+        curr_list_answers_list = list(
+            map(str.strip, [item for sublist in curr_list_answers_list for item in sublist]))
+        return curr_list_answers_list
+
+    def check_date(self, series):
+        for value in series.values:
+            try:
+                datetime.date.fromisoformat(value)
```

Comment:
What is the format of 'value' expected for this check? Examples: '23-01-2023', '23/01/2023', '2023/01/23' don't pass this check.